test/integration/etcd/data.go

```diff
@@ -161,13 +161,6 @@ func GetEtcdStorageDataForNamespace(namespace string) map[schema.GroupVersionRes
 		},
 		// --
 
-		// k8s.io/kubernetes/pkg/apis/batch/v1beta1
```

Comment:
Do not remove this, this ensures that when users request v1beta1 endpoint they're getting batch/v1 cronjob.